Packs/QualysFIM/Integrations/QualysFIM/QualysFIM.py

```diff
@@ -635,9 +634,8 @@ def test_module(client: Client):
         'ok' if test passed, anything else will fail the test.
     """
     try:
-        result = client.incidents_list_test()
-        if result.ok:
-            return 'ok'
+        client.incidents_list_test()  # raises exception if non-okay response
+        return 'ok'
     except Exception as exception:
```

Comment:
not your code, but in line 650 we return a DemistoException - let's change `return` to `raise` and append `from e`

Reply:
Fixed in commit 17a3773a9323138ef3a60ac98b2740e05b1e68d1